tox.ini

```diff
@@ -26,7 +26,7 @@ whitelist_externals = make
 [testenv:py34]
 commands =
     python --version
-    python -m compileall -fq -x 'lib/ansible/module_utils' lib
+    python -m compileall -fq -x 'lib/ansible/module_utils|.tox' lib
```

Comment:
This bit is actually unnecessary, because it's crawling under `./lib/`, not directly in `.`.  I didn't notice that when I was preparing my patch.

In fact, changing `.` to `lib` in the other compileall invocations would perhaps be a better way of fixing this problem.